src/execute/oas3/build-request.js

```diff
@@ -148,7 +148,8 @@ export function applySecurities({request, securities = {}, operation = {}, spec}
         }
         else if (type === 'http') {
           if (schema.scheme === 'basic') {
-            const {username, password} = value
+            const username = value.username
+            const password = typeof value.password === "string" || value.password instanceof String ? value.password : ""
```

Comment:
if the concern here is falsy (eg `null` and `undefined`) values, we can just use logical OR to default to an empty string:

```suggestion
            const password = value.password || ""
```

Reply:
The main concern is undefined, possibly also null, which a user may pass to express the absence of a value. Other problematic cases are unlikely to occur.